go/cmd/dolt/commands/merge.go

```diff
@@ -301,12 +344,14 @@ func constructInterpolatedDoltMergeQuery(apr *argparser.ArgParseResults) (string
 	return interpolatedQuery, nil
 }
 
-// calculateMergeConflicts calculates the count of conflicts that occurred during the merge.
-func calculateMergeConflicts(queryist cli.Queryist, sqlCtx *sql.Context, mergeStats map[string]*merge.MergeStats) (map[string]*merge.MergeStats, bool, int) {
+// calculateMergeConflicts calculates the count of conflicts that occurred during the merge. Returns a map of table name to MergeStats,
+// a bool indicating whether there were any conflicts, and a bool indicating whether calculation was successful.
+func calculateMergeConflicts(queryist cli.Queryist, sqlCtx *sql.Context, mergeStats map[string]*merge.MergeStats) (map[string]*merge.MergeStats, bool, bool) {
```

Comment:
Please name the output fields:

(stats map[string]*merge.MergeStats, conflicts bool, ok bool)

Reply:
Actually, I think you should return the error rather than an ok bool. That way you can remove all of the println calls in this function, and bubble them up to the caller.